paas/client.go

```diff
@@ -172,6 +183,9 @@ func (c *FusemlClient) Apps() error {
 			}
 			routes = strings.Join(ingRoutes, ", ")
 		}
+		if routes == "" {
```

Comment:
Is there a case where `routes` is empty?

Reply:
Ah ... this seems like some relic from testing...